scenarios/kubernetes_e2e.py

```diff
@@ -391,7 +473,7 @@ def main(args):
     if args.test == 'true':
         runner_args.append('--test')
 
-    cluster = cluster_name(args.cluster, os.getenv('BUILD_NUMBER', 0))
+    cluster = cluster_name(args.cluster, os.getenv('BUILD_NUMBER', 0), args.aws_cluster_domain)
```

Comment:
why do we also need aws_cluster_domain? is cluster + bulid_number not unique?

I'm not sure it makes logical sense to have cluster_name for gce jobs be a function of the aws_cluster_domain, even if this is normally false

How about if we append the suffix outside the cluster_name function?